### PR TITLE
Set default value for surefire "trimStackTrace" option to false

### PR DESCRIPTION
**Pull request by Kellindil**
See issue https://github.com/eclipse-tycho/tycho/issues/1551 for details

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -75,6 +75,11 @@ This will then:
 2. enable the generation and embedding of a maven descriptor (optional if you fully manage your pom.xml with all dependencies)
 3. map P2 dependencies to maven dependencies (optional, but most likely required to get good results)
 
+### Default value change for trimStackTrace
+
+tycho-surefire-plugin had set the default value of the trimStackTrace option to true.
+The default will now be aligned with maven-surefire-plugin at false and will need to be manually adjusted for users that really need the stack traces trimmed.
+
```
Comment (laeubi) [inline RELEASE_NOTES.md:82]: Can you add a small snippet that shows how to restore old behavior?

Comment (Kellindil) [reply RELEASE_NOTES.md:82]: sure :)

**Review by laeubi — APPROVED** [review]
Thanks, looks good to me and if we can align with how surefire behaves this is a good change!

**Comment by laeubi** [comment]
GH failures are unelated ...

**Pull request merged**